tools/kmt/kmt.c

```diff
@@ -1532,7 +1528,7 @@ km_rec_keygen_cmn(void *key, uint64_t rid)
         if (len > keybinmin)
             len = keybinmin + (rid % (keybinmax - keybinmin + 1));
 
-        *(uint64_t *)key = rid;
+        *(uint64_t *)key = cpu_to_be64(rid);
```

Comment:
Store kmt binary keys in big-endian so that they will sort lexicographically by record ID.  This means kmt using binary keys won't be able to read a kmt kvdb that was created before this patch.